src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -54,12 +54,18 @@
 public class TimetableSnapshotSourceTest {
 
   private static final LocalDate SERVICE_DATE = LocalDate.parse("2009-02-01");
+  private static final TripUpdate CANCELLATION = new TripUpdateBuilder(
+    "1.1",
+    SERVICE_DATE,
+    CANCELED,
+    ZoneIds.NEW_YORK
+  )
+    .build();
   private TransitModel transitModel;
 
   private final GtfsRealtimeFuzzyTripMatcher TRIP_MATCHER_NOOP = null;
 
-  private final boolean fullDataset = false;
-  private byte[] cancellation;
+  private final boolean FULL_DATASET = false;
```

Comment:
Maybe this should be called `NOT_FULL_DATASET` to make it clearer that we are setting the param to false in the call sites?

Reply:
What does "not full" mean? Partial ? I guess the underlying problem is that the input param i a boolean and not an enum.

Reply:
Thomas is correct. I wanted to turn this boolean into an enum for many years. Do you want me to do that in this PR or another one?